src/main.ts

```diff
@@ -0,0 +1,30 @@
+import { ConfigService } from '@nestjs/config';
+import { NestFactory } from '@nestjs/core';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import { version, name, description } from 'package.json';
+import { AppModule } from './modules/app/app.module';
+import { Logger } from '@nestjs/common';
+
+async function bootstrap(): Promise<void> {
+  const app = await NestFactory.create(AppModule);
+  const configService = app.get(ConfigService);
```

Comment:
```suggestion
  const configService = app.get<ConfigService<IAppConfig, true>>(ConfigService);
```
аби не доводилось строго типізувати змінні як описано нижче